tox.ini

```
@@ -18,7 +18,7 @@ source = src/snowflake/connector
 [tox]
 minversion = 3.7
 envlist = fix_lint,
-          py{36,37,38,39}-{extras,unit,integ,pandas,sso},
+          py{36,37,38,39}-{extras,unit,integ,pandas,sso}-{parallel},
```

Since there's nothing else in the curly braces they don't do anything and could be removed.
```suggestion
          py{36,37,38,39}-{extras,unit,integ,pandas,sso}-parallel,
```

---

We could do here what I commented in the other comment too
```suggestion
          py{36,37,38,39}-{extras,unit-parallel,integ,pandas,sso},
```